docs/intro.md

```diff
@@ -24,3 +27,7 @@ You can always link your own repository from github.
 - Click deploy
 - Approve "Cyclic - Preview" app in github
 - Watch the terminal for your deployment logs
+
+## Language and runtime support
+
```

Comment:
I think node 14 is backwards compatible with node8+

Reply:
Seems true but is that for us to highlight?